node/src/components/network/conman.rs

```diff
@@ -587,8 +661,13 @@ async fn handle_incoming(
                 |dropped| info!(until=?entry.until, justification=%entry.justification, dropped, "peer is still banned")
             );
 
-            // TODO: Send a proper error using RPC client/server here (requires appropriate
-            //       Juliet API). This would allow the peer to update its backoff timer.
+            let peer_error = PeerError::banned(now, entry.until, &entry.justification);
+            tokio::spawn(rpc_server.send_custom_error_and_shutdown(
+                ChannelId::new(0),
```

Comment:
Just to confirm my understanding, this could pretty much be any channel, right?

Reply:
Yes. However errors are not "sent on a channel", since they have precendence over any other message and always single frame. They only require the current frame to be cleared. `juliet` will send nothing after an error has been sent, and will send it as soon as possible.

For this reason it is best to think of the `channel` and `id` fields on the error as "data" fields, indicating where the error occured. Since we do not know in this case, we just set them to 0.